### ISSUE TITLE
cannot import name '_mplcairo' from partially initialized module 'mplcairo'

### ISSUE THREAD

**Issue by DT-one** [matplotlib/mplcairo#57]
Can't run the version command, but can share this (I am trying a reinstall):
```
(base) C:\Users\xxxxx>pip uninstall mplcairo
Found existing installation: mplcairo 0.5
Uninstalling mplcairo-0.5:
  Would remove:
    c:\users\xxxxx\devopssetupprograms\anaconda\lib\site-packages\mplcairo-0.5.dist-info\*
    c:\users\xxxxx\devopssetupprograms\anaconda\lib\site-packages\mplcairo.pth
    c:\users\xxxxx\devopssetupprograms\anaconda\lib\site-packages\mplcairo\*
Proceed (Y/n)? y
  Successfully uninstalled mplcairo-0.5
```

Not sure what extra detail I can provide. Can't even import it. Something looks fundamentally wrong.

```
import matplotlib
matplotlib.use("module://mplcairo.base")

Traceback (most recent call last):

  Cell In[1], line 2
    matplotlib.use("module://mplcairo.base")

  File ~\DevopsSetupPrograms\Anaconda\Lib\site-packages\matplotlib\__init__.py:1249 in use
    plt.switch_backend(name)

  File ~\DevopsSetupPrograms\Anaconda\Lib\site-packages\matplotlib\pyplot.py:342 in switch_backend
    module = importlib.import_module(cbook._backend_module_name(newbackend))

  File ~\DevopsSetupPrograms\Anaconda\Lib\importlib\__init__.py:90 in import_module
    return _bootstrap._gcd_import(name[level:], package, level)

  File <frozen importlib._bootstrap>:1387 in _gcd_import

  File <frozen importlib._bootstrap>:1360 in _find_and_load

  File <frozen importlib._bootstrap>:1310 in _find_and_load_unlocked

  File <frozen importlib._bootstrap>:488 in _call_with_frames_removed

  File <frozen importlib._bootstrap>:1387 in _gcd_import

  File <frozen importlib._bootstrap>:1360 in _find_and_load

  File <frozen importlib._bootstrap>:1331 in _find_and_load_unlocked

  File <frozen importlib._bootstrap>:935 in _load_unlocked

  File <frozen importlib._bootstrap_external>:995 in exec_module

  File <frozen importlib._bootstrap>:488 in _call_with_frames_removed

  File ~\DevopsSetupPrograms\Anaconda\Lib\site-packages\mplcairo\__init__.py:27
    from . import _mplcairo

ImportError: cannot import name '_mplcairo' from partially initialized module 'mplcairo' (most likely due to a circular import) (C:\Users\mu281f\DevopsSetupPrograms\Anaconda\Lib\site-packages\mplcairo\__init__.py)
```

**Comment by anntzer** [comment]
Can you list the contents of `~\DevopsSetupPrograms\Anaconda\Lib\site-packages\mplcairo\`?

**Comment by DT-one** [comment]
```
C:\Users\xxxxx\DevopsSetupPrograms\Anaconda\Lib\site-packages\mplcairo>dir
 Volume in drive C is Windows
 Volume Serial Number is 

 Directory of C:\Users\xxxxxx\DevopsSetupPrograms\Anaconda\Lib\site-packages\mplcairo

09/05/2024  11:14 AM    <DIR>          .
09/05/2024  11:14 AM    <DIR>          ..
09/05/2024  11:14 AM            17,664 base.py
09/05/2024  11:14 AM             1,373 gtk.py
09/05/2024  11:14 AM             1,486 gtk_native.py
09/05/2024  11:14 AM               605 macosx.py
09/05/2024  11:14 AM             2,131 multipage.py
09/05/2024  11:14 AM             2,219 qt.py
09/05/2024  11:14 AM               815 tk.py
09/05/2024  11:14 AM             1,385 wx.py
09/05/2024  11:14 AM               664 _backports.py
09/05/2024  11:14 AM             2,422 _util.py
09/05/2024  11:14 AM               422 _version.py
09/05/2024  11:14 AM             3,234 __init__.py
09/05/2024  11:14 AM    <DIR>          __pycache__
              12 File(s)         34,420 bytes
               3 Dir(s)  90,969,747,456 bytes free
```

**Comment by anntzer** [comment]
The install is broken; the extension module _mplcairo.dll is missing.

**Comment by anntzer** [comment]
Released mplcairo 0.6 with wheels for Python up to 3.13, so you should be able to just install from PyPI.
